test/auth/client_credentials_test.rb

```diff
@@ -0,0 +1,53 @@
+# typed: false
+# frozen_string_literal: true
+
+require_relative "../test_helper"
+
+module ShopifyAPITest
+  module Auth
+    class ClientCredentialsTest < Test::Unit::TestCase
+      def setup
+        super()
+
+        @stubbed_time_now = Time.now
```

Comment:
NIT: This is only used once and I believe `Time.now` is intuitive enough and less verbose to use directly below.